platform/source/lib/ax/platform/application.py

```diff
@@ -299,11 +300,10 @@ def __init__(self, client=None):
             self.client = KubernetesApiClient(use_proxy=True)
         else:
             self.client = client
-        self.ignored_namespaces = frozenset(["kube-system", "default", "axsys", "axuser", "kube-public"])
 
     def list(self):
-        return [x.metadata.name for x in self._get_namespaces().items if x.metadata.name not in self.ignored_namespaces]
+        return [x.metadata.name for x in self._get_namespaces().items]
 
     @retry_unless()
     def _get_namespaces(self):
-        return self.client.api.list_namespace()
+        return self.client.api.list_namespace(label_selector="creator=argo-application")
```

Comment:
Can you verify that the list_namespace function returns an object with empty list for `.items` if no such namespace is found?

Reply:
Done. Confirmed that list_namespace returns an empty list of the label_selector does not match.